iron-request.html

```diff
@@ -286,9 +286,16 @@
           handleAs = 'text';
         }
 
-        // In IE, `xhr.responseType` is an empty string when the response
-        // returns. Hence, caching it as `xhr._responseType`.
-        xhr.responseType = xhr._responseType = handleAs;
+        //placing the following line inside try catch as some browsers like UC doesn't support JSON as response type. It throws SYNTAX_ERROR: DOM EXCEPTION 12
```

Comment:
Can you make this comment more concise? e.g.
`// UC Browser throws exception for JSON response type`
or something under 80 cols?